src/unix/mod.rs

```diff
@@ -16,6 +16,13 @@ pub type cc_t = ::c_uchar;
 pub enum DIR {}
 pub enum locale_t {}
 
+// FIXME: This is technically wrong; idtype_t is specified as a C enum.
+// [ http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/sys_wait.h.html ]
+// However, FFI doesn't currently know how to ABI-match a C enum
+// (rust#28925, rust#34641) and *probably* the underlying type will be
+// c_uint everywhere since all of the enumerators are representable by c_uint.
```

Comment:
I think it's ok to remove this FIXME, I don't think it's possible to fix such a fixme.